### DISCUSSION TITLE
Imported repo's commits are not displayed in contribution calendar graph

### DISCUSSION THREAD

**Discussion by undrcrxwn** [community/community#73438] [Repositories] [answered]
### Select Topic Area

Question

### Body

Recently I migrated from GitLab by [importing](https://github.com/new/import) some repos from there to my GitHub organization. The commits in those repos are properly referencing my profile, but I still can't see any of them in my contribution graph (calendar or whatever it's called). Will my contributions get recounted and displayed on the graph soon or they are just ignored?

For example: [yesterday's commit](https://github.com/crowdparlay/users/commit/3001e644fc6ac02a4a617999aeccff5bd5c2d3b9) and clear graph cells:
![image](https://github.com/community/community/assets/69521267/b72bb378-a503-4886-a73e-38b08e7e326d)

**Comment by esurov** [comment, marked as answer]
Dear @undrcrxwn ,
When you import repositories from GitLab to GitHub, it can take some time for the contributions to appear on your GitHub contribution graph. GitHub's contribution graph is generated and updated periodically, so it may not reflect the imported contributions immediately. The process of recounting and displaying contributions on the graph can take some time, and there might be some delay before you see the imported commits and contributions reflected in your profile.

Here are a few things you can do to ensure that your contributions get counted and displayed on the graph:

1. **Wait for Updates:** Be patient and give GitHub some time to update your contribution graph. Depending on the volume of data, it may take a few hours or even a day for the contributions to be fully counted and displayed.

2. **Check Your Email:** GitHub may send you an email notification once the contributions are updated. This email will confirm that your contributions have been counted and will reflect in your contribution graph.

3. **Verify Repository Ownership:** Make sure that you have the proper ownership and permissions for the repositories in your GitHub organization. Contributions from repositories that you don't own or have access to may not be counted.

4. **Revisit Repository Settings:** Double-check the repository settings, such as whether they are public or private, as this can also impact contribution visibility.

If after waiting for some time and ensuring the above factors, your contributions are still not appearing on the contribution graph, you can consider reaching out to GitHub support for further assistance. They can investigate the issue and provide guidance on how to resolve it.

In most cases, GitHub's systems will catch up and display your contributions on the graph after a delay, but if you continue to experience issues, it's a good idea to seek support from GitHub's official channels.
